examples/notebooks/intro/README.md

```diff
@@ -0,0 +1,34 @@
+# Data Prep Kit Introduction
+
+This is an example featuring some of the features of data prep kit.
+
+## Running the code
+
+The code can be run on either 
+
+1.  Google colab: very easy to run; no local setup needed.
+2.  On your local Python environment.  Here is a quick guide.  You can  find instructions for latest version [here](../../../README.md#-getting-started)
+
+```bash
+conda create -n data-prep-kit -y python=3.11
+conda activate data-prep-kit
+
+# install the following in 'data-prep-kit' environment
```

Comment:
Please add pip3 install data-prep-toollit==0.2.1 as well. Just play it safe so when one of the dependencies change, you have it all covered
```suggestion
# install the following in 'data-prep-kit' environment
pip3 install data-prep-tooklit==0.2.1
```